### PR TITLE
[dialogflow_task_executive] add comments in DialogResponse.msg

### PR DESCRIPTION
**Pull request by mqcmd196**
Adding comments in `DialogResponse.msg`

### CODE REVIEW

File: dialogflow_task_executive/msg/DialogResponse.msg
Comment (k-okada) [inline dialogflow_task_executive/msg/DialogResponse.msg:8]: Please add something like `If dialogflow could not find the action, it returns 'input.unknown' see https://....`, if dialogflow always returns 'input.unknown'. Or is this JSK's rule? I could not find them in https://cloud.google.com/python/docs/reference/dialogflow/latest/google.cloud.dialogflow_v2.types.Intent

Comment (mqcmd196) [reply dialogflow_task_executive/msg/DialogResponse.msg:8]: Not the JSK rule. Dialogflow has `Default Fallback Intent` by default and its action name is `input.unknown`. It is strange that there are no official docs about it, but it can be confirmed both creating new dialogflow agent and complementing

Screenshot from 2022-08-01 09-26-41

Comment (mqcmd196) [reply dialogflow_task_executive/msg/DialogResponse.msg:8]: update the msg doc

**Review by k-okada — CHANGES_REQUESTED** [review]
Please look at https://github.com/ros/executive_smach/blob/noetic-devel/smach_msgs/msg/SmachContainerStatus.msg, https://github.com/ros/common_msgs/blob/noetic-devel/sensor_msgs/msg/Image.msg
 If you hvae comment after variable definistion, start from same column, which looks better.

If you ask me review AFTER all CI passed, then I can push merge button immedieatly. Otherwise, I have to wait CI and sometimes?/usually? I forget about that...

**Comment by mqcmd196** [comment]
Thank you for your review, I understand.

**Pull request merged**
